.github/workflows/frontend_ci.yml

```diff
@@ -0,0 +1,30 @@
+name: learn-github-actions
```

Comment:
Change the name of this github action so it's more appropriate of what it's doing

Reply:
address this ^